src/Functions/array/arrayDotProduct.cpp

```diff
@@ -385,6 +385,35 @@ using FunctionArrayDotProduct = FunctionArrayScalarProduct<DotProduct>;
 
 REGISTER_FUNCTION(ArrayDotProduct)
 {
+    FunctionDocumentation::Description description = R"(
+Returns the dot product of two arrays.
+
+Aliases: `scalarProduct`, `dotProduct`
```

Comment:
The list of aliases is independent metadata. We should have a separate field in `FunctionDocumentation`. Can be done in a separate PR.

Reply:
We can generate these using the `alias_to` column. Will remove.

Reply:
Yepp, just noticed that there is an `alias_to` column already.